nss-day-cohort-52/rare-api-smoky-pythons
# added full CRUD functionality for posts and comments

### Issue ticket(s): #7 #14 #21 #22 #23 

### New file(s): None

### Modified File(s): *See 9 files in the 'files changed' tab of this pull request*

### Modifications:
- Added create, update, and delete methods to the CommentView and PostView classes
- Added a new `is_staff` property to the data object sent back in the Response of the `login_user` function located in `./views/auth.py`
- Changed field names in the Comment model from user_id and post_id to user and post
- Changed the `max_length=` attribute of the content and URL fields on the Post model

### Steps to test:
1. While in the terminal, use git fetch --all in the root directory of this repo
2. Checkout out to al-comments
3. Fire up the server and launch the client
4. Navigate to the PostList component from the navbar
5. Attempt to create a new post, you should be redirected back to the PostList
6. Now try editing a post
7. If successful, try deleting a post
8. Click on the title of a post, you should be directed to the PostDetails component
9. Click on the view comments button and try adding a new comment
10. now try editing that comment
11. Finally, see if you can successfully delete a comment!
